src/sql/foglamp_ddl.sql

```diff
@@ -877,10 +878,16 @@ ALTER TABLE foglamp.schedules OWNER to foglamp;
 CREATE TABLE foglamp.tasks (
   id           uuid                        UNIQUE,                 -- Unique uuid, PK
   process_name character varying(20)       NOT NULL,               -- Name of the task
-  state        smallint                    NOT NULL,               -- State of the task: 1-Running, 2-Complete, 3-Cancelled
+  state        smallint                    NOT NULL,               -- State of the task: 1-Running, 2-Complete, 3-Cancelled Terris:
+                                                                   -- Terris: should there be a "didn't start" state
+                                                                   -- or is this only for tasks that started?
```

Comment:
By didn't start do you mean a task execution was attempted by it failed to run at all? This is a special case of failed, but I see no reason not to have it.